pages/valtio/index.ts

```diff
@@ -0,0 +1,3 @@
+import redirect from '../../utils/redirect'
+
+export default redirect('/valtio/home')
```

Comment:
I think we can do away with this by redirecting paths to their first child, respecting `nav` order.